tests/test_concepts.py

```diff
@@ -809,3 +810,55 @@ def test_requires_paren() -> None:
             ]
         )
     )
+
+
+def test_non_template_requires() -> None:
+    content = """
+      template <class T>
+      struct Payload
+      {
+          constexpr Payload(T v)
+              requires(std::is_pod_v<T>)
```

Comment:
The requires statement gets completely discarded here, whereas in other contexts the requires is retained as a raw Value? Even though we aren't doing anything with the data at the moment, this is inconsistent and could be a problem if we ever decide to do something with the data.

It seems that the best way to deal with this would be to move `raw_requires_post` into Function instead of being in TemplateDecl? Even though that would be a breaking change, since this is mostly unparsed anyways I don't think it's that big of a deal.

Reply:
It isn't discarded immediately, but eventually the TemplateDecl isn't used so it is discarded. If it wasn't discarded, then the `requires(std::is_pod_v<T>)` would be in the testcase's ParsedData somewhere, and it isn't.

When I was looking at this I noticed a simpler way to do this, so I'm going to close this in favor of doing it that way.

Reply:
... which was exactly what you suggested, ha

Reply:
Oh sweet, thank you!